xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-ui/src/main/resources/Mail/Translations.xml

```diff
@@ -67,7 +67,7 @@ Mail.SendMailConfigClass_properties.hint=The list of additional JavaMail propert
 Mail.SendMailConfigClass_sendWaitTime=Send Wait Time
 Mail.SendMailConfigClass_sendWaitTime.hint=The delay to wait between each mail being sent, in milliseconds (defaults to 8 seconds)
 Mail.SendMailConfigClass_discardSuccessStatuses=Discard Success Statuses?
-Mail.SendMailConfigClass_discardSuccessStatuses.hint=Whether mail statuses are kept or not for mail that have been sent successfully (defaults to true)
+Mail.SendMailConfigClass_discardSuccessStatuses.hint=Whether mail statuses are kept or not for mail that has been sent successfully (defaults to true)
```

Comment:
Should the fix be to use "mails" instead of "mail" ? I feel like that was the intent here.

Reply:
I looked into that but mail does not have a plural form.

Reply:
Interesting. Found this: http://www.nytimes.com/2010/05/09/magazine/09FOB-onlanguage-t.html

I still think it would make more sense to use the plural. It looks really strange with singular version IMO. Would be great if some English-native dev or user could help us on this :)

Reply:
I think singular sounds more natural, but we can get a second opinion.